util/src/entities/Config.ts

```diff
@@ -188,6 +189,7 @@ export const DefaultConfigOptions: ConfigValue = {
 		},
 		guild: {
 			maxRoles: 250,
+			maxEmojis: 50, // TODO: max emojis per guild per nitro level
```

Comment:
This is very true, perhaps we could make a class/interface that holds maxEmojis depending on the tier level?